### PR TITLE
HasTerm-instance-for-SimpleApp instance HasTerm SimpleApp

### PR DESCRIPTION
**Pull request by yaitskov**

### CODE REVIEW

File: src/RIO/PrettyPrint.hs [hunk 1]
```diff
@@ -53,6 +58,21 @@ class (HasLogFunc env, HasStylesUpdate env) => HasTerm env where
   useColorL :: Lens' env Bool
   termWidthL :: Lens' env Int
 
+lookupEnvVar :: Text -> SimpleApp -> Maybe Text
+lookupEnvVar name env = M.lookup name $ (getConst . envVarsL Const) env
```
Comment (yaitskov) [inline src/RIO/PrettyPrint.hs:62]: @snoyberg, is there way to avoid global function and reuse code inside multiple methods of a class instance scope?

Comment (snoyberg) [reply src/RIO/PrettyPrint.hs:62]: No, but as long as you don't export it from the module that's fine.

File: src/RIO/PrettyPrint.hs [hunk 2]
```diff
@@ -53,6 +58,21 @@ class (HasLogFunc env, HasStylesUpdate env) => HasTerm env where
...
+
+instance HasStylesUpdate SimpleApp where
+  stylesUpdateL = lens (\_ -> parseStylesUpdateFromString "") const
+
+instance HasTerm SimpleApp where
+  useColorL =
+    lens ((== Just "truecolor") . lookupEnvVar "COLORTERM")
+    const
+  termWidthL =
+    let parseColumn = maybe 80 id . join . fmap (readMaybe . T.unpack)
```
Comment (snoyberg) [inline src/RIO/PrettyPrint.hs:72]: This is triggering a linter failure by not using `fromMaybe`

**Review by snoyberg — CHANGES_REQUESTED** [review]

**Comment by mpilgrem** [comment]
I am going to close this pull request for the same reason that I have closed #6.

**Pull request closed without merging**